templates/postgresql.yml.erb

```diff
@@ -33,6 +33,10 @@ bootstrap:
 <% end -%>
     - encoding: <%= @initdb_encoding %>
     - locale: <%= @initdb_locale %>
+<% if @initdb_encryption_command -%>
+    - encryption-key-command: >-
```

Comment:
Why not put the value on this line? The notation to do multiline string is I think what is used here but the value is wrapped in quotes so not sure that makes sense.  Unless `>-` does something else?

Reply:
+1 since the value is a string we want `key: "string"` instead of introducing harder to read yaml.

Reply:
Updated.  Thanks.